.github/workflows/integration_test.yaml

```diff
@@ -5,7 +5,7 @@ on:
 
 jobs:
   integration-tests:
-    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    uses: nrobinaubertin/stable-operator-workflows/.github/workflows/integration_test.yaml@main
```

Comment:
Is this intended?
Is the long term plan to use this?

Reply:
Whoops, good catch.

I decided to use my stable fork because of some issues with operator-workflows on my last PRs. But those are resolved now